src/generic/modal-dropzone/ModalDropzone.jsx

```diff
@@ -93,7 +102,9 @@ const ModalDropzone = ({
                   onProcessUpload={handleSelectFile}
                   inputComponent={inputComponent}
                   accept={accept}
```

Comment:
I don't think the accept is formatted the correct way because it is not only allowing PDFs to be selected for the textbooks page. I was able to select a `.docx` file and was shown the invalid file type error.

Reply:
Fixed